aspnetcore/fundamentals/minimal-apis/test-min-api.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: Test Minimal API apps
+title: Test Minimal API applications
 author: rick-anderson
-description: Unit and integration tests in Minimal API apps
+description: Unit and integration tests in Minimal API applications
```

Comment:
@Rick-Anderson I thought we preferred `apps` over `applications` due to shortness?

Reply:
Oops